site/src/contexts/auth/permissions.tsx

```diff
@@ -11,13 +13,17 @@ export const checks = {
 	viewUpdateCheck: "viewUpdateCheck",
 	viewExternalAuthConfig: "viewExternalAuthConfig",
 	viewDeploymentStats: "viewDeploymentStats",
+	readWorkspaceProxies: "readWorkspaceProxies",
 	editWorkspaceProxies: "editWorkspaceProxies",
 	createOrganization: "createOrganization",
 	editAnyOrganization: "editAnyOrganization",
 	viewAnyGroup: "viewAnyGroup",
 	createGroup: "createGroup",
 	viewAllLicenses: "viewAllLicenses",
-} as const;
+	viewNotificationTemplate: "viewNotificationTemplate",
+} as const satisfies Record<string, string>;
+
+type PermissionType = keyof typeof checks;
```

Comment:
this is technically incorrect, and only working because all of the key value pairs are symmetrical. what you really want is...

```suggestion
type PermissionType = typeof checks[keyof typeof checks];
```